### DISCUSSION TITLE
Authentication, Authorisation, Teams, Security Policies

### DISCUSSION THREAD

**Discussion by thclark** [django-guardian/django-guardian#830] [Announcements]
Hi All, creating a discussion thread for this.

We @maintainers need to determine roles/ teams and a few quick policies to ensure secure access. We should implement practices/policies for:

- security (eg how people report vulnerabilities)
- authorisation (eg how collaorators/maintainers access django-guardian
- authentication (what teams we should have and what roles each team has)

Please do brainstorm here!

**Comment by thclark** [comment]
I made a couple of basic changes to start with that should be no-brainers:

1. I removed the `Owners` team because it was creating confusion (it didn't actually convey owner privilege). The number of owners should be small enough to [view and manage directly here](https://github.com/orgs/django-guardian/people) (external collaborators please note that some members may choose to remain private so you may not be seeing an exhaustive list).

2. I required 2FA authentication for all organisation members as an essential basic security precaution.

---

*A couple of apologies*

First a quick apology to @brianmay - I deleted the reviewers team before seeing your message [here](https://github.com/django-guardian/django-guardian/issues/603#issuecomment-2424210822), as I was clearing up and saw you had duplicate permissions so thought it was older. I'll reinstate... but what other teams do you think we need with which roles?

 @BonaFideIT please could you enable 2FA on your GitHub account? I'll then be able to reinstate your membership of the organisation.

**Reply by BonaFideIT** [reply]
2FA should be enabled now.
